libs/src/api/rewards.js

```diff
@@ -454,11 +452,15 @@ class Rewards extends Base {
 
     try {
       const response = await axios(request)
+      // if attestation is successful, 'result' represents a signature
+      // otherwise, 'result' is false
+      // - there may or may not be a value for `needs` if the attestation fails
+      // - depending on whether the user can take an action to attempt remediation
       const { result, needs } = response.data
 
-      if (needs) {
+      if (!result) {
         logger.error(`Failed to get AAO attestation: needs ${needs}`)
```

Comment:
This log should probably check for the existence of needs then?

Reply:
thought about it. was lazy. lol let's do it